xc7/tests/oserdes_routing/CMakeLists.txt

```diff
@@ -0,0 +1,23 @@
+add_file_target(FILE oserdes_routing.v SCANNER_TYPE verilog)
+add_file_target(FILE dummy.pcf)
+
+add_fpga_target(
+  NAME oserdes_routing
+  BOARD basys3-bottom
+  SOURCES oserdes_routing.v
+  INPUT_IO_FILE dummy.pcf
+  EXPLICIT_ADD_FILE_TARGET
+  )
+
+add_vivado_target(
+  NAME oserdes_routing_vivado
+  PARENT_NAME oserdes_routing
+  # The fasm diff test is disabled as it is expected to fail for now. Should
+  # be re-enabled when #1258 is fixed
+  DISABLE_DIFF_TEST
```

Comment:
I believe the DIFF test should be enabled, as the purpose of this test is to verify this behavior. This way I could take over this PR which can be merged as soon as the const issue is fixed.

Reply:
Ok, I've enabled the diff test. Should fail in Vivado with net conflict between <const 0> and <const 1>